DevEdu.API/DevEdu.DAL/Repositories/StudentAnswerOnTaskRepository.cs

```diff
@@ -0,0 +1,95 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Data;
+using System.Text;
+using System.Data.SqlClient;
+using Dapper;
+using DevEdu.DAL.Models;
+using System.Linq;
+
+namespace DevEdu.DAL.Repositories
+{
+    public class StudentAnswerOnTaskRepository : BaseRepository, IStudentAnswerOnTaskRepository
+    {
+        public StudentAnswerOnTaskRepository()
+        {
+
+        }
+
+        public void DeleteStudentAnswerOnTaskDto(StudentAnswerOnTaskDto studentResponse)
+        {
+            _dbconnection.Execute(
+                "dbo.Task_Student_Delete",
```

Comment:
please create consts for all the stored procedures' names (like we did it in the lesson)